### PR TITLE
Toxin lovers don't throw up.

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request
Toxin lovers don't throw up.

## Why It's Good For The Game
Some toxins tox you and make you throw. But you like tox!

## Changelog
:cl:
add: Toxin lovers don't throw up.
/:cl:

### CODE REVIEW

**Comment by TerraGS** [comment]
Slimepeople should throw up healing chems

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -429,7 +429,7 @@
 	return ..()
 
 /mob/living/carbon/proc/vomit(lost_nutrition = 10, blood = FALSE, stun = TRUE, distance = 1, message = TRUE, toxic = FALSE, harm = TRUE, force = FALSE)
-	if(HAS_TRAIT(src, TRAIT_NOHUNGER) && !force)
+	if(HAS_TRAIT(src, TRAIT_NOHUNGER) && HAS_TRAIT(src, TRAIT_TOXINLOVER) && !force)
```
Comment (JJRcop) [inline code/modules/mob/living/carbon/carbon.dm]: OR
```suggestion
	if((HAS_TRAIT(src, TRAIT_NOHUNGER) || HAS_TRAIT(src, TRAIT_TOXINLOVER)) && !force)
```

Comment (Dennok) [reply code/modules/mob/living/carbon/carbon.dm]: Oh yes. Thank.

**Comment by Dennok** [comment]
> Slimepeople should throw up healing chems

Bui now it dont intended. It don't throw anything. And liver dont filter healing chems.

**Comment by PKPenguin321** [comment]
You misspelled "lovers" as "lowers" in your changelog (and PR title and body)

**Pull request merged**